src/main/docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG APP_DIR=/opt/owasp/dependency-track/
 ARG DATA_DIR=/data/
 ARG UID=1000
 ARG GID=1000
-ARG WAR_FILENAME=dependency-track-apiserver.jar
+ARG WAR_FILENAME=dependency-track-bundled.jar
```

Comment:
Whoops, this change shouldn't be here. 👀 Please revert.

Reply:
done in https://github.com/elastic-pangolin/dependency-track/commit/17d7af47bc9d4bb1b74ce40b1a5af5e770e9c0be